docs/DocumentationStyleGuide/accessibility.rst

```diff
@@ -0,0 +1,94 @@
+Accessibility
+=============
+
+The moja global community strives towards accessibility in our software
+products and even in the documentation. Through accessibility, we strive
+for successful access to information and resources among people who
+might find it challenging to access them otherwise. We aim to bring
+contributors with limited ability to the forefront of our community to
+have them contribute and access our documentation easily.
+
+The general guidelines for creating accessible documentation pieces are:
+
+-  Maintain consistency across the documentation. Both in style and
+   visuals by employing a fixed hierarchy.
+-  Avoid language or word usage that reeks of bias or a disregard for a
+   group of people.
+-  Focus on the audience that you are writing for and follow a
+   conversation style of documentation.
+-  Define acronyms, abbreviations, prerequisites initially and never
+   assume the technical competency of the reader.
+-  Use alternative text for multimedia elements wherever possible, and
+   be conscious of the usage of visual elements.
+
+Image accessibility
+-------------------
+
+-  Validate that the images, diagrams, graphs, icons possess a
+   meaningful alternative-text description for a meaningful equivalent.
+-  Use alternative text to describe what the image, digraph, graph or
+   icon does, rather than what it looks like.
+-  All images, diagrams, graphs, icons should contain a short image
```

Comment:
I think 'caption' is a good word here. Absolutely agree they are mandatory. It should tell the reader what the key takeway of the image is, so that the reader can examine the image and ensure they derive the same conclusion, cementing the correct understanding.